SBAT.md

```diff
@@ -140,29 +140,39 @@ value associated with the data currently stored in that variable.
 
 #### Generation-Based Revocation Scenarios
 
-Products (**not** vendors, a vendor can have multiple products or even pass a
+**Products** (**not** vendors, a vendor can have multiple products or even pass a
 product from one vendor to another over time) are assigned a name. Product
 names can specify a specific version or refer to the entire product family. For
 example mydistro and mydistro,12.
 
-Components that are used as a link in the UEFI Secure Boot chain of trust are
+**Components** that are used as a link in the UEFI Secure Boot chain of trust are
 assigned names. Examples of components are shim, GRUB, kernel, hypervisors, etc.
 
+Below is an example of a product and component, both in the same `sbat.csv` file. `sbat`
+is a component, as it is a link in the UEFI Secure Boot chain of trust.
```

Comment:
I see in my email you left a comment requesting that I switch this to `'sbat' defines the version of the format of the revocation metadata itself`however I don't see that comment here. (Was it deleted? I also see in your lgtm comment you wanted me to address something you flagged.)
Does that comment still apply?
I had my specific wording originally because the word "component" seems to be used in a specific way in this document, as you mentioned, and I wanted it to be clear what the word "component" referred to. But I do get that components is an overloaded word. It's entirely possible I didn't understand what you meant by "component" in this document. Is `sbat` a component as defined above? Or is it just "the version of the format of the revocation metadata itself"? If that is true, should I use a different example to illustrate what a "component" is?
Here's a guess as to what I could switch it to, to clarify the concern you had with the word being overloaded:
```
**Component** as defined here refers to a component that is committed to a UEFI boot services variable, as opposed to a component loaded from a file in a filesystem. They are used as a link in the UEFI Secure Boot chain of trust and are assigned names...
```

I'm more than happy to rewrite it in the way you prefer, I just want to make sure that is still what you want, as that comment seems to be gone.